integration-tests/deployment/devenv/don.go

```diff
@@ -0,0 +1,302 @@
+package devenv
+
+import (
+	"context"
+	"fmt"
+	"math/big"
+	"strconv"
+	"strings"
+
+	"github.com/AlekSi/pointer"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/hashicorp/go-multierror"
+	chainselectors "github.com/smartcontractkit/chain-selectors"
+
+	clclient "github.com/smartcontractkit/chainlink/integration-tests/client"
+	"github.com/smartcontractkit/chainlink/integration-tests/deployment"
+	nodev1 "github.com/smartcontractkit/chainlink/integration-tests/deployment/jd/node/v1"
+	"github.com/smartcontractkit/chainlink/integration-tests/deployment/jd/shared/ptypes"
+	"github.com/smartcontractkit/chainlink/integration-tests/web/sdk/client"
+)
+
+const (
+	NodeLabelType      = "type"
+	NodeLabelBootstrap = "bootstrap"
+	NodeLabelPlugin    = "plugin"
```

Comment:
I feel like NodeLabelType is not like NodeLabelBootstrap - isn't `type` a key and `bootstrap` a value in some sort of key-value mapping somewhere?